pkg/metrics/tags_test.go

```diff
@@ -33,7 +34,23 @@ var testM = stats.Int64(
 	"A metric just for tests",
 	stats.UnitDimensionless)
 
-func TestNewStatsReporterCtxErrors(t *testing.T) {
+func register(t *testing.T) {
+	if err := view.Register(
+		&view.View{
+			Description: "Number of pods autoscaler wants to allocate",
+			Measure:     testM,
+			Aggregation: view.LastValue(),
+			TagKeys:     append(CommonRevisionKeys, ResponseCodeKey, ResponseCodeClassKey),
+		}); err != nil {
+		t.Fatalf("Failed to register view: %v", err)
+	}
+}
```

Comment:
I wonder if it'd be cool to return unregister as `cancelFunc` from here.

Reply:
Kinda cool? 😂

Reply:
that's what cool kids do in Go :)